repose-aggregator/components/filters/valkyrie-authorization/src/main/resources/META-INF/schema/config/valkyrie-authorization.xsd

```diff
@@ -82,6 +82,22 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="pass-non-dedicated-tenant" type="xs:boolean" use="optional" default="false">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        If enabled (true), the Valkyrie filter WILL NOT ensure that the tenant ID presented is a
+                        dedicated tenant ID (i.e., starts with "hybrid:"). If it is not, the Valkyrie filter will
+                        pass the request to the next filter in the filter chain.
```

Comment:
Heh, that's funny. I actually didn't like the way this read, but couldn't think of a better way of phrasing it at the time. I think I even tried using the word verify, but that doesn't quite work as a drop-in replacement. The Valkyrie filter will attempt to verify that the tenant ID is a dedicated tenant ID regardless, what this attribute changes is the behavior if verification fails.

What about this:
> If enabled (true), the Valkyrie filter WILL pass the request to the next filter in the filter chain IF the tenant ID presented is NOT a dedicated tenant ID (i.e., starts with "hybrid:").
> If disabled (false), the Valkyrie filter WILL NOT pass the request to the next filter in the filter chain IF the tenant ID presented is NOT a dedicated tenant ID (i.e., starts with "hybrid:"). Instead, the request will be considered unauthorized.

Or, if you'd like to change some words up, go ahead and leave them in a comment. I have no problem changing this documentation.